content/platforms/kubernetes/quick-start.md

```diff
@@ -0,0 +1,164 @@
+
+ The operator currently runs within single namespace and is scoped to operate only on cluster CRD's in that namespace. This allows a multi-tenant deployment of the operator supporting different versions in different namespaces.
+
+## Installing the operator
+
+The definition and reference materials for the operator are on
+[GitHub](https://github.com/RedisLabs/redis-enterprise-k8s-docs) and the
+implementation is published as a Docker container. The operator
+definitions are packaged as a single generic YAML file called
+[bundle.yaml](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/bundle.yaml).
+
+You need to ensure you pull the correct version of the bundle. The version tags
+can be found by checking the [releases on Github](https://github.com/RedisLabs/redis-enterprise-k8s-docs/releases)
+or via the GitHub API.
+
+The bundle for the latest release can be downloaded via `curl` using the following:
+
+```
+
+VERSION=`curl --silent https://api.github.com/repos/RedisLabs/redis-enterprise-k8s-docs/releases/latest | jq -r .tag_nam`
+curl --silent -O https://raw.githubusercontent.com/RedisLabs/redis-enterprise-k8s-docs/$VERSION/bundle.yaml
+```
+
+or you can replace `VERSION` in the above with a specific release tag you desire.
+
+
+If you inspect this YAML file, you will find definitions for the various items
+described in the previous section.
+
+You can install the operator with a single apply command:
+
+```
+kubectl apply -f bundle.yaml -n my-namespace
+```
+
+You should see a result similar to:
+
+```
+role.rbac.authorization.k8s.io/redis-enterprise-operator created
+serviceaccount/redis-enterprise-operator created
+rolebinding.rbac.authorization.k8s.io/redis-enterprise-operator created
+customresourcedefinition.apiextensions.k8s.io/redisenterpriseclusters.app.redislabs.com configured
+customresourcedefinition.apiextensions.k8s.io/redisenterprisedatabases.app.redislabs.com configured
+deployment.apps/redis-enterprise-operator created
+```
+
+## Verifying the operator is running
+
+You can verify the operator is running in your namespace checking the deployment:
+
+```
+kubectl get deployment -l name=redis-enterprise-operator -n my-namespace
+```
+
+You should see a result similar to:
+
+```
+NAME                        READY   UP-TO-DATE   AVAILABLE   AGE
+redis-enterprise-operator   1/1     1            1           0m36s
+```
+
+## Testing the operator
+
+A cluster is created by creating a custom resource with the kind "RedisEnterpriseCluster"
+that contains the specification of the cluster option. See the
+[Redis Enterprise Cluster API reference](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/redis_enterprise_cluster_api.md)
+for more information on the various options available.
+
+You can test the operator by creating a minimal cluster by following this procedure:
+
+1. Create a file called `simple-cluster.yaml` that defines a Redis Enterprise
+   cluster with three nodes:
+
+    ```
```

Comment:
I think you should add a pre-requisites reference since this deployment requires at least 3 nodes with at least 4cpu/GB each whereas the Operator deployment is very modest.

Reply:
Added a note and example of configuring the node resources.